jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -31,6 +31,14 @@
             description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
     scm:
         - git:
+            url: https://github.com/kubernetes/test-infra
```

Comment:
the top answer links to https://wiki.jenkins-ci.org/display/JENKINS/Multiple+SCMs+Plugin, which says that its deprecated and that we should use pipelines instead (https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Plugin).